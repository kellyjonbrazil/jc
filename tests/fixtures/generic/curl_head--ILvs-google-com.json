[{"_type":"request","_request_method":"HEAD","_request_uri":"/","_request_version":"HTTP/1.1","host":"google.com","user-agent":"curl/8.1.2","accept":["*/*"]},{"_type":"response","_response_version":"HTTP/1.1","_response_status":301,"_response_reason":["Moved Permanently"],"location":"http://www.google.com/","content-type":"text/html; charset=UTF-8","content-security-policy-report-only":["object-src 'none';base-uri 'self';script-src 'nonce-w0if96oCu3EX7tRPyppOPg' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp"],"date":"Sun, 04 Feb 2024 22:25:09 GMT","expires":"Tue, 05 Mar 2024 22:25:09 GMT","cache-control":["public","max-age=2592000"],"server":["gws"],"content-length":219,"x-xss-protection":"0","x-frame-options":"SAMEORIGIN","date_epoch_utc":1707085509,"expires_epoch_utc":1709677509},{"_type":"request","_request_method":"HEAD","_request_uri":"/","_request_version":"HTTP/1.1","host":"www.google.com","user-agent":"curl/8.1.2","accept":["*/*"]},{"_type":"response","_response_version":"HTTP/1.1","_response_status":200,"_response_reason":["OK"],"content-type":"text/html; charset=ISO-8859-1","content-security-policy-report-only":["object-src 'none';base-uri 'self';script-src 'nonce-NZmPRDFJDBUjfIdQ12Z3HA' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp"],"p3p":"CP=\"This is not a P3P policy! See g.co/p3phelp for more info.\"","date":"Sun, 04 Feb 2024 22:25:09 GMT","server":["gws"],"x-xss-protection":"0","x-frame-options":"SAMEORIGIN","transfer-encoding":["chunked"],"expires":"Sun, 04 Feb 2024 22:25:09 GMT","cache-control":["private"],"set-cookie":["1P_JAR=2024-02-04-22; expires=Tue, 05-Mar-2024 22:25:09 GMT; path=/; domain=.google.com; Secure","AEC=Ae3NU9Nm-_YoxvDgLc8KacFFXJ7YeR8_4xABOi_6TFPHkH3czLzKUMi2_eU; expires=Fri, 02-Aug-2024 22:25:09 GMT; path=/; domain=.google.com; Secure; HttpOnly; SameSite=lax","NID=511=D1Wo-3cLpM2WxRhuncNJmxkzx96hYr_1QWM5fVPzgPWf03fLZh_MfZEfA0YeSu443m6byeo7l0N9GqFJBxxGMghbMM9p89yQ6ELzgL9N4j31Ejjxj9BxFJ8ccmD794UlzMGAC3PKHTUDUpMRCJ9OOcgiJ8f67JE23iVrWFFkP2w; expires=Mon, 05-Aug-2024 22:25:09 GMT; path=/; domain=.google.com; HttpOnly"],"date_epoch_utc":1707085509,"expires_epoch_utc":1707085509}]
