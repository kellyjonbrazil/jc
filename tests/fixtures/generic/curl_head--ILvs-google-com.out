*   Trying [2607:f8b0:4002:c11::71]:80...
* Connected to google.com (2607:f8b0:4002:c11::71) port 80 (#0)
> HEAD / HTTP/1.1
> Host: google.com
> User-Agent: curl/8.1.2
> Accept: */*
> 
< HTTP/1.1 301 Moved Permanently
< Location: http://www.google.com/
< Content-Type: text/html; charset=UTF-8
< Content-Security-Policy-Report-Only: object-src 'none';base-uri 'self';script-src 'nonce-w0if96oCu3EX7tRPyppOPg' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp
< Date: Sun, 04 Feb 2024 22:25:09 GMT
< Expires: Tue, 05 Mar 2024 22:25:09 GMT
< Cache-Control: public, max-age=2592000
< Server: gws
< Content-Length: 219
< X-XSS-Protection: 0
< X-Frame-Options: SAMEORIGIN
< 
* Connection #0 to host google.com left intact
* Issue another request to this URL: 'http://www.google.com/'
*   Trying [2607:f8b0:4002:c03::69]:80...
* Connected to www.google.com (2607:f8b0:4002:c03::69) port 80 (#1)
> HEAD / HTTP/1.1
> Host: www.google.com
> User-Agent: curl/8.1.2
> Accept: */*
> 
< HTTP/1.1 200 OK
< Content-Type: text/html; charset=ISO-8859-1
< Content-Security-Policy-Report-Only: object-src 'none';base-uri 'self';script-src 'nonce-NZmPRDFJDBUjfIdQ12Z3HA' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp
< P3P: CP="This is not a P3P policy! See g.co/p3phelp for more info."
< Date: Sun, 04 Feb 2024 22:25:09 GMT
< Server: gws
< X-XSS-Protection: 0
< X-Frame-Options: SAMEORIGIN
< Transfer-Encoding: chunked
< Expires: Sun, 04 Feb 2024 22:25:09 GMT
< Cache-Control: private
< Set-Cookie: 1P_JAR=2024-02-04-22; expires=Tue, 05-Mar-2024 22:25:09 GMT; path=/; domain=.google.com; Secure
< Set-Cookie: AEC=Ae3NU9Nm-_YoxvDgLc8KacFFXJ7YeR8_4xABOi_6TFPHkH3czLzKUMi2_eU; expires=Fri, 02-Aug-2024 22:25:09 GMT; path=/; domain=.google.com; Secure; HttpOnly; SameSite=lax
< Set-Cookie: NID=511=D1Wo-3cLpM2WxRhuncNJmxkzx96hYr_1QWM5fVPzgPWf03fLZh_MfZEfA0YeSu443m6byeo7l0N9GqFJBxxGMghbMM9p89yQ6ELzgL9N4j31Ejjxj9BxFJ8ccmD794UlzMGAC3PKHTUDUpMRCJ9OOcgiJ8f67JE23iVrWFFkP2w; expires=Mon, 05-Aug-2024 22:25:09 GMT; path=/; domain=.google.com; HttpOnly
< 
* Connection #1 to host www.google.com left intact
